sys/arvo.hoon

```diff
@@ -328,11 +330,12 @@
           ==
       ?~  pru
         ~&  [%swim-lost lal (symp +>-.hil)]
-        [[~ p.sew] q.sew]
-      =^  pro  p.sew  (need pru)
-      =^  moz  p.sew  (~(slap wa p.sew) pro [%limb %p])
-      =^  vem  p.sew  (~(slap wa p.sew) pro [%limb %q])
-      [(said moz) (soar vem)]
+        [~ [vase.vane worm.vane]]
+      =^  pro  worm.vane  (need pru)
+      =^  moz  worm.vane  (~(slap wa worm.vane) pro [%limb %p])
+      =^  vem  worm.vane  (~(slap wa worm.vane) pro [%limb %q])
```

Comment:
While you're in here, i think it's a little odd to dispatch off of `p` and `q` faces in the vane invocation product. Wouldn't it fine to just `+slot:wa` for the head or tail?

Reply:
Switched this to `+slot:wa`, along with removing the superfluous `%p`/`%q` from vane's `+take` and `+call` arms.